supermarq/features.py

```diff
@@ -0,0 +1,163 @@
+from typing import Union
+
+import cirq
+import networkx as nx
+import numpy as np
+import qiskit
+import supermarq as sm
+
+
+def compute_communication(circuit: Union[cirq.Circuit, qiskit.circuit.QuantumCircuit]) -> float:
+    """
+    communication feature = circuit's average qubit degree / degree of a complete graph
+
+    Input
+    -----
+    circ : A cirq or qiskit quantum circuit
+    """
+
+    if isinstance(circuit, cirq.Circuit):
+        circ = sm.converters.cirq_to_qiskit(circuit)
+    else:
+        circ = circuit
+
+    N = circ.num_qubits
+    dag = qiskit.converters.circuit_to_dag(circ)
+    dag.remove_all_ops_named("barrier")
+
+    G = nx.Graph()
+    for op in dag.two_qubit_ops():
+        q1, q2 = op.qargs
+        G.add_edge(q1.index, q2.index)
+
+    degree_sum = sum([G.degree(n) for n in G.nodes])
+
+    return degree_sum / (N * (N - 1))
+
+
+def compute_liveness(circuit: Union[cirq.Circuit, qiskit.circuit.QuantumCircuit]) -> float:
+    """
+    liveness feature = sum of all entries in the liveness matrix / (num_qubits * depth)
```

Comment:
Change docstring to be in the style of Google's recommendation: https://google.github.io/styleguide/pyguide.html